### PR TITLE
Updates logentry_args.md

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

`logentry_args.md` defines argument names to be used in messages, but there is a limitation in the message model. An argument name can only be used once in the same message. This PR specifies a method to extend the name for such cases.

This PR also:

* Updates the headline
* Adds an introduction and some specification
* Splits the table into two tables, defined and not yet defined names
* Removes old list not relevant

## How to test this PR

This is document only, but the new method is already used in #995.

### CODE REVIEW

File: docs/logentry_args.md [hunk 1]
```diff
@@ -1,25 +1,74 @@
-# Types of information in Zonemaster log objects
+# Argument names for test case messages
```
Comment (mattias-p) [inline docs/logentry_args.md:1]: `Argument names` -> `Arguments`?

This document also concerns itself with the argument values.

File: docs/logentry_args.md [hunk 2]
```diff
@@ -1,25 +1,74 @@
...
+cases, e.g. [Basic.pm], and translated in the PO files, e.g. [fr.po] and [sv.po].
+The arguments are used in the messages, and are there put within curly brackets,
+e.g. as `{ns}`.
+
+When a message is created or updated only argument names defined in this document
+should be used. If there is defined argument name that can be used for the
+message then a new argument name must be defined and this document is to be
+updated.
+
+## Multiple argument of the same type
```
Comment (mattias-p) [inline docs/logentry_args.md:17]: argument -> arguments

**Review by matsduf — COMMENTED** [review]
When working on the text, I realized that "message" and "argument" could be used consistently. I have done further updates.

**Comment by matsduf** [comment]
@mattias-p and @vlevigneron, can you (re-)review this?

**Review by mattias-p — APPROVED** [review]

**Comment by matsduf** [comment]
@vlevigneron, does this looks good to you?

**Pull request merged**